sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EChangeFeedITest.scala

```diff
@@ -178,6 +187,159 @@ class SparkE2EChangeFeedITest
     rowCount shouldEqual 2
   }
 
+  "spark change feed micro batch (incremental)" can "be used to copy data to another container" in {
+    val cosmosEndpoint = TestConfigurations.HOST
+    val cosmosMasterKey = TestConfigurations.MASTER_KEY
+
+    val sourceContainer = cosmosClient.getDatabase(cosmosDatabase).getContainer(cosmosContainer)
+
+    val testId = UUID.randomUUID().toString.replace("-", "")
```

Comment:
'-' is allowed character for container name. Are you seeing errors in presence of '-'

Reply:
No - just copy & paste and the fact that I mostly use Guid.ToString("N") in .Net whihc reflects above. I have removed it here in this test case.